self-api/src/main/java/com/selfxdsd/api/Stars.java

```diff
@@ -13,4 +13,11 @@
      *  was successful or not.
      */
     boolean add();
+
+    /**
+     * Checks if a repository is starred by the authenticated user.
+     * @return True when starred or false if not or something went wrong
+     * (http error like unauthenticated user etc...).
```

Comment:
@criske "Check if the authenticated user added the star". No need to mention the repo, it's already clear that these Stars belong to a repo.